tests/functional/heketi/test_block_volumes_heketi.py

```diff
@@ -310,18 +315,16 @@ def test_create_max_num_blockhostingvolumes(self):
                           % (free_space_available, default_bhv_size))
 
         # Create BHV's
+        count = 1
         while free_space_available > default_bhv_size:
-            block_host_create_info = heketi_volume_create(
-                self.heketi_client_node, self.heketi_server_url,
-                default_bhv_size, json=True, block=True)
-            if block_host_create_info["id"] not in existing_bhv_list.keys():
-                new_bhv_list.append(block_host_create_info["id"])
-            self.addCleanup(
-                heketi_volume_delete, self.heketi_client_node,
-                self.heketi_server_url, block_host_create_info["id"],
-                raise_on_error=False)
-
-            free_size = block_host_create_info["blockinfo"]["freesize"]
+            bhv_name = "{}-bhv-{}".format(prefix, count)
+            bhv_info = self.create_heketi_volume_with_name_and_wait(
+                bhv_name, default_bhv_size,
+                raise_on_cleanup_error=False, json=True, block=True)
+            if bhv_info["id"] not in existing_bhv_list.keys():
+                new_bhv_list.append(bhv_info["id"])
```

Comment:
```suggestion
            if bhv_info.get("id") not in existing_bhv_list.keys():
                new_bhv_list.append(bhv_info.get("id"))
```

Reply:
Please check PR title, it's meant to fix issues related logic. Suggested change won't affect any steps/logic which will fix the issue.

Reply:
Yes, do you mean suggested conventional changes are not valid with this PR?

Reply:
What do you mean by conventional ? We switched to use `get` for dict in sametime back long time after this TC. And it's not mandatory use `get`. So this won't affect the code.

Reply:
Sure, Vaibhav! It is conventional as `.get` can be used to avoid KeyError. Well, it does not fix the original logic.